pyjade/ext/django/loader.py

```diff
@@ -66,7 +66,7 @@ def load_template(self, template_name, template_dirs=None):
             key = '-'.join([template_name, hashlib.sha1('|'.join(template_dirs)).hexdigest()])
 
         
-        if settings.DEBUG or key not in self.template_cache:
+        if settings.DEBUG or settings.TEMPLATE_DEBUG or key not in self.template_cache:
```

Comment:
Since `TEMPLATE_DEBUG` is [deprecated as of Django 1.8](https://docs.djangoproject.com/en/1.8/ref/settings/#template-debug), perhaps
```python
getattr(settings, 'TEMPLATE_DEBUG', False)
```
would play nicer with new django projects where it might not be defined?